sources/api/schnauzer/src/helpers.rs

```diff
@@ -160,6 +160,12 @@ mod error {
             template: String,
         },
 
+        #[snafu(display("Invalid template value, {} in template {}", message, template))]
+        InvalidTemplateValueCustomMessage {
+            message: &'static str,
+            template: String,
+        },
+
```

Comment:
Is this enum variant being used anywhere? If it's not being used, we should remove it.

Reply:
Correct, this one is not used -- it is a remnant of a previous approach to the PR (pre-latest-refactor).
Removing..